### PR TITLE
HW2 (mw3071) rebased to fix conflicts

### PR DESCRIPTION
**Pull request by mw3071**
Added Github readme and profile link

Create columbia-discord-bot.md

Create discord-py.md

moved to projects/python folder

Delete discord-py.md since already exists in /python

Create discordpy.md

### CODE REVIEW

File: mw3071.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<h1>Open Source Development</h1>
```
Comment (timkpaine) [inline mw3071.md:1]: move file to `students/mw3071.md`, use markdown format for whole file

File: mw3071.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+Name: Matthew Wang
+
+UNI: mw3071
+
+GitHub: [link](https://github.com/mw3071)
+
+GitHub README: [link](https://github.com/mw3071/project-proposals-s2023/blob/main/README.md)
+
+<h2>Project Proposal</h2>
+Columbia Discord Bot
```
Comment (timkpaine) [inline mw3071.md:12]: markdown link to file

File: mw3071.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+<h2>Project Contributions</h2>
+Discordpy
```
Comment (timkpaine) [inline mw3071.md:15]: markdown link to file

File: projects/python/columbia-discord-bot.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+projects/columbia-discord-bot.md
```
Comment (timkpaine) [inline projects/python/columbia-discord-bot.md:1]: remove this line

File: projects/python/columbia-discord-bot.md [hunk 5]
```diff
@@ -0,0 +1,6 @@
...
+# Columbia University Discord Bot
+
```
Comment (timkpaine) [inline projects/python/columbia-discord-bot.md:3]: link to project repository

File: projects/python/discordpy.md [hunk 6]
```diff
@@ -0,0 +1,13 @@
+# Columbia University Discord Bot
+
+## Description
+This discord bot will help students in our discord server find other students in the same class or have experiences with that class so that they can seek homework help, recommendations, or reviews of that class.
+Another feature would be to allow students to know what computers are avaliable at the Wallach Gaming Lounge and be able to reserve space or see when computers will be free.
+
+projects/discordpy.md
```
Comment (timkpaine) [inline projects/python/discordpy.md:7]: remove this line

**Review by timkpaine — CHANGES_REQUESTED** [review]

**Comment by mw3071** [comment]
created pull request for the file in students/mw3071.md:
https://github.com/ColumbiaOSS/project-proposals-s2023/pull/431

**Pull request merged**